dipy/sims/voxel.py

```diff
@@ -288,7 +299,265 @@ def multi_tensor(gtab, mevals, S0=100, angles=[(0, 0), (90, 0)],
+    based on multicompartmental models which assumes that tissue are described
+    by impermeable diffusion compartments characterized by their only diffusion
+    tensor.
+
+    Parameters
+    -----------
+    gtab : GradientTable
+    mevals : array (K, 3)
+        eigenvalues of the diffusion tensor for each individual compartment
+    S0 : float
+        Unweighted signal value (b0 signal).
+    angles : array (K,2) or (K,3)
+        List of K tensor directions of the diffusion tensor of each compartment
+        in polar angles (in degrees) or unit vectors
+    fractions : float
+        Percentage of the contribution of each tensor. The sum of fractions
+        should be equal to 100%.
+    snr : float
+        Signal to noise ratio, assuming Rician noise.  If set to None, no
+        noise is added.
+
+    Returns
+    --------
+    dt : (6,)
+        elements of the diffusion tensor.
+    kt : (15,)
+        elements of the kurtosis tensor
+    S : (N,) ndarray
+        Simulated signal based on the dki model.
+            Note: This simulated signal ignores the DW taylor accumulantes 
+                  higher than the 4th order. This simulations are useful to 
+                  test dki fit procedures, since tensors fitted on S have to 
+                  match the diffusion and kurtosis tensor produced here. For 
+                  simulations of S with the higher accumulates please refer to 
+                  the multi_tensor function.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from dipy.sims.voxel import multi_tensor_dki
+    >>> from dipy.data import get_data
+    >>> from dipy.core.gradients import gradient_table
+    >>> from dipy.io.gradients import read_bvals_bvecs
+    >>> fimg, fbvals, fbvecs = get_data('small_101D')
+    >>> bvals, bvecs = read_bvals_bvecs(fbvals, fbvecs)
+    >>> gtab = gradient_table(bvals, bvecs)
+    >>> mevals=np.array(([0.0015, 0.0003, 0.0003],[0.0015, 0.0003, 0.0003]))
+    >>> e0 = np.array([1, 0, 0.])
+    >>> e1 = np.array([0., 1, 0])
+    >>> dt, kt, S =  multi_tensor_dki(gtab, mevals)
+    
+    References
+    ----------
+    .. [1] R. Neto Henriques et al., "Exploring the 3D geometriy of the 
+           diffusion kurtosis tensor - Impact on the development of robust 
+           tractography procedures and novel biomarkers", NeuroImage (2015) 
+           111, 85-99.
+    """
+                             
+    if np.round(np.sum(fractions), 2) != 100.0:
+        raise ValueError('Fractions should sum to 100')
+
+    fractions = [f / 100. for f in fractions]
+
+    S = np.zeros(len(gtab.bvals))
+    
+    angles = np.array(angles)
+    if angles.shape[-1] == 3:
+        sticks = angles
+    else:
+        sticks = [sphere2cart(1, np.deg2rad(pair[0]), np.deg2rad(pair[1]))
+                  for pair in angles]
+        sticks = np.array(sticks)
+    
+    # computing a 3D matrice containing the individual DT components 
+    mD=np.zeros((len(fractions),3,3))
```

Comment:
PEP8: spaces around operators and after comma: 

    mD = np.zeros((len(fractions, 3, 3))